src/perf-tests/perftest-stack-airline/.gitignore

```diff
@@ -0,0 +1,7 @@
+!jest.config.js
+*.d.ts
+node_modules
+
+# CDK asset staging directory
```

Comment:
Add the `.js` files as you accidentally included them in git - `npm run build` will transpile TS to JS